src/index.js

```diff
@@ -394,9 +394,7 @@ export function ConfirmMultisigAddress({keystore, network, bip32Path, multisig,
 export function ConfigAdapter({ KEYSTORE, jsonConfig }) {
```

Comment:
`generateColdcardConfig` and `parseColdcardConfig` have been moved out of the config class now into their own functions in the `coldcard.js` file. I left this Adapter class as is since this was the existing interface and I didn't want to interfere with any other downstream dependencies, but for someone who wants to functionally interact with a coldcard config, that capacity now exists, which has also simplified the config struct/class considerably.